language/compiler/src/lib.rs

```diff
@@ -89,18 +89,12 @@ impl<'a> Compiler<'a> {
     }
 
     /// Compiles the code and arguments into a `Program` -- the bytecode is serialized.
-    pub fn into_program(mut self, args: Vec<TransactionArgument>) -> Result<Program> {
+    pub fn into_program(mut self, args: Vec<TransactionArgument>) -> Result<Script> {
```

Comment:
should we call this into_script? and fix the comment as well?

Reply:
The reason why I didn't rename it to into_script is because we've already have `input_script` in the scope.